src/EA.Weee.Web/Areas/Admin/ViewModels/Aatf/AatfEditDetailsViewModel.cs

```diff
@@ -0,0 +1,52 @@
+﻿namespace EA.Weee.Web.Areas.Admin.ViewModels.Aatf
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Core.DataStandards;
+    using EA.Weee.Core.Shared;
+
+    public class AatfEditDetailsViewModel
+    {
+        public AatfEditDetailsViewModel()
+        {
```

Comment:
Can we add some unit tests to check that the properties of the view model have the correct attributes, mainly the regular expressions and required fields :)

Reply:
Added